docs/authentication/social-connections/facebook.mdx

```diff
@@ -8,7 +13,7 @@ Adding social connection with Facebook to your app with Clerk is simple - you on
 
 To make the development flow as smooth as possible, Clerk uses  preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed.
 
-For production instances, you will need to create your own developer account with **Facebook** and generate your own **App ID** and **App Secret**.
+For production instances, you will need to create your own developer account with Facebook and generate your own **App ID** and **App Secret**.

```

Comment:
Keeping it consistent with how "Facebook" is referenced throughout the doc. It is not bolded throughout.